### DISCUSSION TITLE
Response stops streaming

### DISCUSSION THREAD

**Discussion by sam1am** [danny-avila/LibreChat#2641] [Troubleshooting]
### What happened?

Frequently when a model response is streaming, the UI seems to stop updating and get stuck. I believe the streaming is still happening in the background because if you wait for it to finish and refresh the page, the response will be there. 

### Steps to Reproduce

1. This seems to happen regardless of language model provider. 
2. I am using Firefox, which may be relevant. 

### What browsers are you seeing the problem on?

Firefox

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Issues like this seem to be a network connectivity issue between the UI and backend, or between the backend and the remote service.

It can't be reliably reproduced, unless I introduce throttling to the network.

**Comment by danny-avila** [comment]
For example, this is my remote instance and it's not having this issue:

https://github.com/danny-avila/LibreChat/assets/110412045/6a2516e0-2d6b-4d0a-a037-4a01c58b3682

**Reply by fuegovic** [reply]
It is very hard to reproduce, some endpoints seem to do it more often then others... It was mentionned at least once on discord:
[discord #general](https://discord.com/channels/1086345563026489514/1086345563026489517/1228390524097134593)

**Comment by aswny** [comment]
@sam1am @fuegovic did you find any solution to this?

I am facing similar issue with a setup of AWS EC2 VM on which LibreChat & LiteLLM are hosted. The streaming just gets stuck by the end of response and I have to refresh everytime to see the full response and use LibreChat again.

**Reply by aswny** [reply]
@danny-avila I know it's hard to reproduce, but if you have any helpful pointers to try out on this setup it would be very helpful. Thanks.

**Reply by danny-avila** [reply]
@aswny 

1. update librechat to latest commit
2. try with and without liteLLM, compare results
3. rule out connection issues between you and the AWS EC2 VM (closer region, better connection)
